tetrateci/create_istio_release.sh

```diff
@@ -5,13 +5,20 @@ set -o pipefail
 
 BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
+export BUILD_WITH_CONTAINER=0
```

Comment:
Move this right below line 37 and a comment on why this is needed. (Refer to my previous comment made in this regard.)